plugin/src/main/java/org/owasp/benchmarkutils/score/parsers/ThunderScanReader.java

```diff
@@ -62,15 +62,11 @@ private TestCaseResult toTestCaseResult(
 
         tcResult.setCWE(
                 figureCwe(vulnerabilityType.name, vulnerability.function, vulnerability.filename));
-        int testcasenum = testNumber(vulnerability.filename);
-        if (testcasenum > 0) {
-            tcResult.setNumber(testcasenum);
-            tcResult.setCategory(vulnerabilityType.name);
-            tcResult.setConfidence(1);
-            tcResult.setEvidence(lineNumber(vulnerability));
-            return tcResult;
-        }
-        return null; // Finding not in a test case
+        tcResult.setNumber(testNumber(vulnerability.filename));
```

Comment:
Why is the logic changed here? If the finding is NOT in a test case, we don't want to create a finding for it.

Reply:
I did not figure out that logic, I reversed the code, now it is the same as the previous one. 
Anyway, it is not consistent with all the parsers. In some case, there is no this type of logic that returns null when the test case is not useful. I suggest to review the code and define a specific logic for all the parsers.